src/components/EpisodePreview/EpisodePreview.jsx

```diff
@@ -0,0 +1,147 @@
+import React, { useContext } from "react";
+import {
+	ACTIONS,
+	EpisodesContext as Episodes,
+	TrailersContext as Trailers
+} from "COMPONENTS/Episodes";
+import UTILS from "SHARED/utils.js";
+
+export default function EpisodePreview(props){
+
+	// CONTEXT
+	// ----------------------------
+	const {
+		state: {
+			activeIndex,
+			overlayActive,
+			episodeCount
+		},
+		dispatch
+	} = useContext(Episodes);
+
+	const {
+		prevTrailer: {
+			src: prevTrailerSrc
+		},
+		nextTrailer: {
+			src: nextTrailerSrc
+		}
+	} = useContext(Trailers);
+
+
+	// RENDER
+	// ----------------------------
+	const { 
+		episodeId,
+		index,
+		number: episodeNo,
+		title,
+		thumbnail: {
+			src: thumbSrc,
+			description: thumbDesc
+		},
+		trailer: {
+			src: trailerSrc
+		}
+	} = props;
+
+	// handle visibility of active episode and episode preview overlay
+	const isActiveEpisode 	= activeIndex === index;
+	const isOverlayActive   = isActiveEpisode && overlayActive;
+	const hidden 			= !isOverlayActive;
+
+	const isFirst 			= index === 0;
+	const isLast  			= index === episodeCount - 1;
+
+	const formattedTitle 	= UTILS.convertToSafeString(title, "-");
+	const previewId 	 	= `${formattedTitle}-preview`;
```

Comment:
Looks good on my screen 😕 
![image](https://user-images.githubusercontent.com/8885493/64724736-fc20c880-d4ca-11e9-929e-0c4143f34062.png)

Reply:
ahhh, I just checked and it's because you've mixed tabs & spaces in the wrong way

- tabs for indentation
- spaces for formatting

![image](https://user-images.githubusercontent.com/3370135/64727628-27a6b180-d4d1-11e9-8d4f-bf7a0d808400.png)

By using tabs in formatting, the formatting goes wild whenever you change your tab-width. Your default is the same as mine- namely 4 spaces, but if I set my tab-width to 2 spaces...

![image](https://user-images.githubusercontent.com/3370135/64727693-502eab80-d4d1-11e9-9694-c84900abb8bc.png)

So yeah - tabs to indent; spaces for formatting.